presto-main/src/test/java/io/prestosql/security/TestAccessControlManager.java

```diff
@@ -460,130 +457,5 @@ public void checkCanSelectFromColumns(SystemSecurityContext context, CatalogSche
     private static class DenyConnectorAccessControl
             implements ConnectorAccessControl
     {
-        @Override
-        public void checkCanSelectFromColumns(ConnectorSecurityContext context, SchemaTableName tableName, Set<String> columnNames)
-        {
-            denySelectColumns(tableName.toString(), columnNames);
-        }
-
-        @Override
-        public void checkCanCreateSchema(ConnectorSecurityContext context, String schemaName)
-        {
-            throw new UnsupportedOperationException();
-        }
-
-        @Override
-        public void checkCanDropSchema(ConnectorSecurityContext context, String schemaName)
-        {
-            throw new UnsupportedOperationException();
```

Comment:
> Simplify

Not sure why it was doing this, but it was throwing UnsupportedOperationException not DenyException.
I understand TACM was not supposed to reach this method, and if it reached, a test would fail with UOE.

Do you know why UOE here?

Reply:
I don't know either. This class is only used in this test, so I don't think we need to think here that deeply.

Reply:
> I understand TACM was not supposed to reach this method, and if it reached, a test would fail with UOE.

This is assumption sounds very fragile (almost like mocking). Even it was true we should not maintain that.